package org.xzc.msg.exception;

/**
 * 重复的用户组
 * @author xzchaoo
 *
 */
public class RepeatedUserGroupException extends KnownException {

	public RepeatedUserGroupException() {
		super();
		// TODO Auto-generated constructor stub
	}

	public RepeatedUserGroupException(String message, Throwable cause, boolean enableSuppression,
			boolean writableStackTrace) {
		super( message, cause, enableSuppression, writableStackTrace );
		// TODO Auto-generated constructor stub
	}

	public RepeatedUserGroupException(String message, Throwable cause) {
		super( message, cause );
		// TODO Auto-generated constructor stub
	}

	public RepeatedUserGroupException(String message) {
		super( message );
		// TODO Auto-generated constructor stub
	}

	public RepeatedUserGroupException(Throwable cause) {
		super( cause );
		// TODO Auto-generated constructor stub
	}

}
